lib/ansible/modules/apt.py

```diff
@@ -1139,6 +1139,7 @@ def main():
         DEBIAN_FRONTEND='noninteractive',
         DEBIAN_PRIORITY='critical',
         LANG=locale,
+        LANGUGE=locale,
```

Comment:
Shoulnd't this be `LANGUAGE`. Also I would have assumed `LANG`  or `LC_*` affects this as well. Have you verified this actually fixed your issue?

Reply:
Yes, sorry for the typo. I will try to add a test on the weekend. It does fix the issue locally, as does adding 
```
environment:
        LANGUAGE: en_US.utf8
```
to the play